railties/lib/rails/commands/dbconsole/dbconsole_command.rb

```diff
@@ -99,6 +99,10 @@ def environment
       Rails.respond_to?(:env) ? Rails.env : Rails::Command.environment
     end
 
+    def connection
+      @options[:connection] || "primary"
```

Comment:
Yes, totally fair point and thus, this will be printed running `--help`, thanks !